modules/bq-log-alerting/use-cases/non_allowlisted_services.sql

```diff
@@ -0,0 +1,51 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+WITH log_table AS (
+  SELECT
+    receiveTimestamp,
+    timestamp,
+    protopayload_auditlog.requestMetadata.callerIp AS callerIp,
+    protopayload_auditlog.authenticationInfo.principalEmail AS principalEmail,
+    logName,
+    log.resource.labels.project_id AS projectId,
+    REGEXP_EXTRACT(authInfo.resource, r '^services/([^/]*)') AS serviceName
+  FROM
+    `${project}.raw_logs.cloudaudit_googleapis_com_activity_*` log,
```

Comment:
Missing `AS` in

`    `${project}.raw_logs.cloudaudit_googleapis_com_activity_*` AS log,`